src/PKSim.Core/Mappers/PKValuesToPKAnalysisMapper.cs

```diff
@@ -31,15 +31,21 @@ public class PKValuesToPKAnalysisMapper : IPKValuesToPKAnalysisMapper
          _displayUnitRetriever = displayUnitRetriever;
       }
 
-      public PKAnalysis MapFrom(DataColumn dataColumn, PKValues pkValues, PKParameterMode mode, string moleculeName)
+      public PKAnalysis MapFrom(DataColumn dataColumn, PKValues pkValues, PKParameterMode mode, string moleculeName, bool forPopulation)
```

Comment:
I Am not sure I understand the forPopulation
I think we should filter also for individual as well no ?
/cc @PavelBal

Reply:
Actually, you are right, and in the process it will be easier to read